Add a placeholder for the Portfile to patch in the real path.
--- edbrowse/doc/man-edbrowse-freebsd.1.orig	2022-03-12 23:41:26.000000000 -0600
+++ edbrowse/doc/man-edbrowse-freebsd.1	2022-06-21 02:28:11.000000000 -0500
@@ -129,7 +129,7 @@
 .Em userguide.html
 ,type:
 .Pp
-.Dl edbrowse /usr/local/share/doc/edbrowse/usersguide.html
+.Dl edbrowse @DOCDIR@/usersguide.html
 .Sh AUTHORS
 .Nm
 was primarily written by Karl Dahlke <eklhad@comcast.net>.
